docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,495 @@
+corner of the picture above). It is commonplace to have this branch represent
+the latest, clean version of this repository's content. For the purposes of your
+PR, let's create a `feature` branch with a name indicative of your changes. In
+my case, it is `emonddr-doc-changes`.
+
+Click on the `Clone or download` button
+
+![submit_pr_create_feature_branch_1.png](./imgs/submit_pr_create_feature_branch_1.png)
+
+This brings up tiny dialog with different choices : Clone with SSH, Use HTTPS,
+Open in Desktop, or Download Zip.
+
+In my case, I will leave it as `Clone with SSH`, and click on the
+`copy to clipboard` icon on the right side of the repository url (we are going
+to paste this value in a terminal window soon).
+
+![submit_pr_create_feature_branch_2.png](./imgs/submit_pr_create_feature_branch_2.png)
+
+Open a terminal window, and navigate to the directory where you want to clone
+the repository. In my case, this is `/Users/dremond/git` .
+
+To create the `feature` branch, run:
+
+```
+git clone { repository url from clipboard }
+cd loopback-next
+git checkout -b { your feature branch }
+```
+
+To install the package dependencies, run
+
+```
+npm install
+```
+
+#### 3. Make changes locally
+
+Whether you are contributing to code or documentation, make all your changes
+inside in the local feature branch directory.
+
+To ensure your files are free of formatting, syntax, and logical/execution
+errors, run :
+
+```
+npm run lint:fix && npm run test
+```
+
+#### 4. Commit changes to the local feature branch
+
+When you are pleased with the work you have done on the local copy of your
+feature branch, you will want to `stage` your changes in preparation for a
+`commit`.
+
+To find out which files have been modified, added, deleted, and also what is
+staged or unstaged, run:
+
+```
+git status
+```
+
+To stage all untracked files, run:
+
+```
+git add --all
+```
+
+To stage one untracked file at a time, run:
+
+```
+git add  { relative path to file from root directory }
+```
+
+When you are pleased with your staged changes, it is time to create a `commit`
+and give it a message.
+
+Our commit messages are formatted according to
+[Conventional Commits](https://conventionalcommits.org/).
+
+Please read the
+[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+guidelines to correctly format your commit messages.
+
+To help with abiding by the rules of commit messages, please use the
+`commitizen` tool mentioned in the documentation above. This means will we use
+`git cz` instead of `git commit`. Install `commitizen` by running
+`npm install -g commitizen`.
+
+In my case, I want the commit message to look like this:
+
+```
+docs: tutorial on how to submit a pull request to lb4
+
+This tutorial shows contributors step-by-step instructions on how to submit a pull request (PR) to LoopBack v4
+
+```
+
+To start `commitizen`, run :
```

Comment:
ditto with `run :` -> `run:` and anywhere else you might have that space (don't want to bombard you with nitpick comments :-) )